event-app-ui/src/components/Popup/Popup.jsx

```diff
@@ -15,21 +17,105 @@ export default function Popup(props) {
   const [location, setLocation] = useState("");
   const [startTime, setStartTime] = useState("");
   const [endTime, setEndTime] = useState("");
+  const [users, setUsers] = useState([]);
+  const [selectedUsers, setSelectedUsers] = useState([]);
+  const [gaps, setGaps] = useState([]);
+  const [error, setError] = useState();
+  const [isChecked, setIsChecked] = useState(false);
 
-  const { setIsOpen } = props;
-  const { username } = props;
+  const dateConversionConfig = {
+    year: "2-digit",
+    month: "2-digit",
+    day: "2-digit",
+    hour: "numeric",
+    minute: "numeric",
+    hour12: true,
+  };
+
+  const { username, setIsOpen } = props;
 
-  const eventInput = () => {
-    axios.post(`http://localhost:3000/event_popup`, {
+  const eventInput = async (e) => {
+    const response = await axios.post(`http://localhost:3000/event_popup`, {
```

Comment:
when js object have same key and value name you can use es6 short-hand

more you can find here https://attacomsian.com/blog/javascript-object-property-shorthand